DevOps pipeline should include all error information when there are errors in Verify-ChangeLog.ps1

As an example, PR https://github.com/Azure/azure-sdk-for-net/pull/29133 executed the [net - translation - ci](https://dev.azure.com/azure-sdk/public/_build/results?buildId=1622468&view=results) pipeline which failed with:
> ##[error]eng\Directory.Build.Common.targets(38,5): Error : No entry for version [1.0.0] found in the ChangeLog [D:\a\_work\1\s\sdk\translation\Azure.AI.Translation.Document\CHANGELOG.md].

The error when the script is executed locally has more information:
>Write-Error: C:\repo\dotnet\eng\common\scripts\ChangeLog-Operations.ps1:394
Line |
 394 |      LogError "The changelog entry has the following sections with no  …
     |      ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     | The changelog entry has the following sections with no content (Other Changes). Please ensure to
     | either remove the empty sections or add content to the section.

Displaying the full error in DevOps will be more helpful.

--- a/eng/Directory.Build.Common.targets
+++ b/eng/Directory.Build.Common.targets
@@ -35,7 +35,7 @@
     <Exec ContinueOnError="true" ConsoleToMSBuild="true" StandardOutputImportance="High" Command="$(PowerShellExe) -NoProfile -NonInteractive -executionpolicy Unrestricted -File $(ValidateReleaseNotesScriptPath) -ChangeLogLocation $(ChangeLogPath) -VersionString $(_VersionInProject)">
       <Output TaskParameter="ExitCode" PropertyName="SetReleaseNotesErrorCode" />
     </Exec>
-    <Error Condition="'$(SetReleaseNotesErrorCode)' != '0'" Text="ChangeLog Verification failed at [$(ChangeLogPath)]." />
+    <Error Condition="'$(SetReleaseNotesErrorCode)' != '0'" Text="ChangeLog verification failed for [$(ChangeLogPath)]." />
   </Target>
 
   <!--Run Aggregate Updates to Source -->

--- a/eng/Directory.Build.Common.targets
+++ b/eng/Directory.Build.Common.targets
@@ -32,10 +32,10 @@
       <ValidateReleaseNotesScriptPath Condition=" '$(ValidateReleaseNotesScriptPath)'=='' ">$(MSBuildThisFileDirectory)common/scripts/Verify-ChangeLog.ps1</ValidateReleaseNotesScriptPath>
       <ChangeLogPath>$([MSBuild]::NormalizeDirectory($(MSBuildProjectDirectory)/../))CHANGELOG.md</ChangeLogPath>
     </PropertyGroup>
-    <Exec ContinueOnError="true" ConsoleToMSBuild="true" StandardOutputImportance="Low" Command="$(PowerShellExe) -NoProfile -NonInteractive -executionpolicy Unrestricted -File $(ValidateReleaseNotesScriptPath) -ChangeLogLocation $(ChangeLogPath) -VersionString $(_VersionInProject)">
+    <Exec ContinueOnError="true" ConsoleToMSBuild="true" StandardOutputImportance="High" Command="$(PowerShellExe) -NoProfile -NonInteractive -executionpolicy Unrestricted -File $(ValidateReleaseNotesScriptPath) -ChangeLogLocation $(ChangeLogPath) -VersionString $(_VersionInProject)">
       <Output TaskParameter="ExitCode" PropertyName="SetReleaseNotesErrorCode" />
     </Exec>
-    <Error Condition="'$(SetReleaseNotesErrorCode)' != '0'" Text="No entry for version [$(_VersionInProject)] found in the ChangeLog [$(ChangeLogPath)]. " />
+    <Error Condition="'$(SetReleaseNotesErrorCode)' != '0'" Text="ChangeLog Verification failed at [$(ChangeLogPath)]." />
   </Target>
 
   <!--Run Aggregate Updates to Source -->

